config/locales/de.yml

```diff
@@ -0,0 +1,204 @@
+  profiles:
+    edit:
+      title: Bearbeite Profil
+      hide_email: Verberge E-Mail in öffentliches Profil
+      reset_password:
+        title: Zurücksetzen des Passworts
+        text: "Brauchst du ein neues Passwort? Kein Problem. %{link}"
+        link_text: Jetzt ein neues Passwort zuschicken lassen.
+      api_access:
+        title: API-Zugang
+        key_is: "Dein API-Schlüssel ist %{key}."
+        credentials: "Wenn du %{gem_commands_link} über das Terminal oder über das CMD verwenden möchtest, dann brauchst du die %{gem_credentials_file}-Datei, die du mit dem folgenden Befehl erzeugen kannst:"
+        link_text: Gem-Befehle
+        reset: Zurücksetzen meines API-Schlüssels
+        confirm_reset: Bist du sicher? Das kann nicht rückgängig gemacht werden.
+
+  rubygems:
+    dependencies:
+      header: "%{title} Abhängigkeiten"
+    edit:
+      title: "Bearbeite Links"
+      subtitle: "für %{gem}"
+      other_fields_notice: Alle anderen Felder können nur durch eine gepushte, aktualisierte Version deines Gems bearbeitet werden.
+    index:
+      title: 'Gems'
+      downloads: Downloads
+    show:
+      not_hosted_notice: Dieses Gem wird nicht aktuell auf RubyGems.org gehostet.
+      install: installieren
+#      yanked_notice: "This gem has been yanked, and it is not available for download directly or for other gems that may have depended on it."
+      authors_header: Autoren
+      downloads_for_this_version: "Für diese Version"
+      owners_header: Besitzern
+      links:
+        header: Links
+        home: Startseite
+        code: Quellcode
+        docs: Dokumentation
+        wiki: Wiki
+        mail: Mailingliste
+        bugs: Bug Tracker
+        download: Download
+        subscribe: Abonniere
+        unsubscribe: Storniere
+        report_abuse: Missbrauch melden
+        badge: Abzeichen
+        rss: RSS
+      versions_header: Versionen
+      bundler_header: Gemfile
+      show_all_versions: "Zeige alle Versionen (%{count} total)"
+      licenses_header: Lizenz
+      no_licenses: k.A.
+      requirements_header: Anforderungen
+      sha_256_checksum: SHA 256-Prüfsumme
+      copy_to_clipboard: In die Zwischenablage kopieren
+      copied: Kopiert!
+      required_ruby_version: Erforderliche Ruby-Version
+
+  searches:
+    show:
+      subtitle: "für %{query}"
+      exact_match: Exakte Übereinstimmung
+
+  sessions:
+    new:
+      forgot_password: "Passwort vergessen?"
+
+  stats:
+    index:
+      total_downloads: Downloads insgesamt
+      total_gems: Gems insgesamt
+      total_users: Gems insgesamt
+      all_time_most_downloaded: Am meisten Heruntergeladen in der gesamten Zeit
+
+  users:
+    new:
+      have_account: "Hast du schon ein Account?"
+
+  versions:
+    index:
+      title: "Alle Versionen von %{name}"
+      versions_since: "%{count} Versionen seit %{since}"
+      not_hosted_notice: Dieses Gem wird nicht gerade von RubyGems.org gehostet.
+    version:
+#      yanked: "yanked"
```

Comment:
@sferik @arthurnn: :+1: I commented the word "yanked" out and squashed the commit, too:

```bash
Windows PowerShell
Copyright (C) 2015 Microsoft Corporation. Alle Rechte vorbehalten.

C:\Users\Suriyaa\Documents\GitHub> git clone https://github.com/SuriyaaKudoIsc/rubygems.org.git
Cloning into 'rubygems.org'...
remote: Counting objects: 31017, done.
remote: Compressing objects: 100% (10/10), done.
remote: Total 31017 (delta 4), reused 0 (delta 0), pack-reused 31007
Receiving objects: 100% (31017/31017), 8.26 MiB | 3.22 MiB/s, done.
Resolving deltas: 100% (20330/20330), done.
Checking connectivity... done.
C:\Users\Suriyaa\Documents\GitHub> cd rubygems.org
C:\Users\Suriyaa\Documents\GitHub\rubygems.org [master]> git checkout patch-2
Branch patch-2 set up to track remote branch patch-2 from origin.
Switched to a new branch 'patch-2'
C:\Users\Suriyaa\Documents\GitHub\rubygems.org [patch-2]> git rebase -i master
[detached HEAD 55f6190] Create de.yml
 Author: Suriyaa Kudo <SuriyaaKudoIsc@users.noreply.github.com>
 Date: Sun Dec 6 13:08:35 2015 +0100
 1 file changed, 203 insertions(+)
 create mode 100644 config/locales/de.yml
Successfully rebased and updated refs/heads/patch-2.
C:\Users\Suriyaa\Documents\GitHub\rubygems.org [patch-2]> git push -f origin patch-2
Counting objects: 5, done.
Delta compression using up to 4 threads.
Compressing objects: 100% (5/5), done.
Writing objects: 100% (5/5), 3.65 KiB | 0 bytes/s, done.
Total 5 (delta 3), reused 0 (delta 0)
To https://github.com/SuriyaaKudoIsc/rubygems.org.git
 + e8501d5...55f6190 patch-2 -> patch-2 (forced update)
C:\Users\Suriyaa\Documents\GitHub\rubygems.org [patch-2]>
```